Core/DISET/private/BaseClient.py

```diff
@@ -315,9 +315,9 @@ def _connect( self ):
           self.__discoverURL()
           return self._connect()
         else:
-          return S_ERROR( "Can't connect to %s: %s" % ( self.serviceURL, retVal ) )
```

Comment:
This was by far the worst! The others were just losing information or badly formatting (str(exception) is very bad)